Dockerfile.ros

```diff
@@ -0,0 +1,85 @@
+FROM ghcr.io/tiiuae/fog-ros-baseimage-builder:v2.1.0 AS builder
+ARG BUILD_GSTREAMER
+# TODO: not sure how many of these deps are actually needed for building. at least this:
+# libusb-1.0-0-dev
+# gstreamer-1.0 (<-- but not sure which package brings this)
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
+    libusb-1.0-0-dev \
+    nlohmann-json3-dev \
+    libopencv-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN if [ -n "$BUILD_GSTREAMER" ]; then \
+    apt-get update -y && apt-get install -y --no-install-recommends \
+    libgstreamer1.0-0 \
+    libgstreamer-plugins-base1.0-dev \
+    libgstreamer-plugins-bad1.0-dev \
+    libgstreamer-plugins-good1.0-dev \
+    libgstrtspserver-1.0-dev \
+    libgstreamer-plugins-base1.0-0 \
+    libgstreamer-plugins-good1.0-0 \
+    gstreamer1.0-x \
+    && rm -rf /var/lib/apt/lists/*; \
+    fi
+
+RUN curl https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/yolo-v4-tiny-tf_openvino_2021.4_6shave.blob \
+    -o /tmp/yolo-v4-tiny-tf_openvino_2021.4_6shave.blob
+
+COPY . /main_ws/src/
+
+RUN if [ -n "$BUILD_GSTREAMER" ]; then \
+    mv /main_ws/src/debian/control.em.gstreamer /main_ws/src/debian/control.em; \
+    fi
+
+# this:
+# 1) builds the application
+# 2) packages the application as .deb in /main_ws/
+RUN BUILD_GSTREAMER=$BUILD_GSTREAMER /packaging/build.sh
+
+#  ▲               runtime ──┐
+#  └── build                 ▼
+
+FROM ghcr.io/tiiuae/fog-ros-baseimage:v2.1.0
+ARG BUILD_GSTREAMER
+
+RUN mkdir /depthai_configs
+COPY --from=builder /main_ws/src/params /depthai_configs/.
+COPY --from=builder /tmp/yolo-v4-tiny-tf_openvino_2021.4_6shave.blob /depthai_configs/.
+
+VOLUME /depthai_configs
+ENV DEPTHAI_PARAM_FILE /depthai_configs/parameters.yaml
+
+ENTRYPOINT [ "/entrypoint.sh" ]
+
+COPY entrypoint.sh /entrypoint.sh
+
+COPY --from=builder /main_ws/ros-*-depthai-ctrl_*_amd64.deb /depthai.deb
+# The multi-arch build has the unnecessary line "/etc/init.d/udev start" for udev rules.
+# Not needed in ubuntu OS baseimage
+RUN sed -i '/\/etc\/init.d\/udev start/d' /entrypoint.sh
```

Comment:
The entrypoint file contains "udev start" but then the sed command deletes that line?

Can't the "udev start" be removed from the container then?

Reply:
`Dockerfile.ros` is an legacy format of building. We don't use that for multiarch, it is left there to have the reference and have ability to run legacy format. That's why it is removing it, because it doesn't require the udev start command.

However, the Dockerfile does not remove the sed line as it needs it.

Reply:
Oh sorry I didn't realize this was not used.

I get the idea of leaving it as a reference.

I'm worried about maintenance: by leaving it here we are either:

a) maintaining something that will not be used.
b) (more likely) leaving this here to rot and then people getting confused about mismatches in `Dockerfile` vs `Dockerfile.ros`
c) risk people looking at the wrong file (happened to my stupid ass now :) )

if this file is for reference purposes, can't we at top of `Dockerfile` leave a comment like this:

```dockerfile
# this file before multi-arch work looked like this:
# https://github.com/tiiuae/depthai_ctrl/blob/d51b9765e7f017bd062f872d8bc53ed7eb5a1b78/Dockerfile.ros
```